src/core/NotePlayHandle.cpp

```diff
@@ -527,13 +527,13 @@ void NotePlayHandle::updateFrequency()
 	if (m_instrumentTrack->m_microtuner.enabled())
 	{
 		// custom key mapping and scale: get frequency from the microtuner
-		const float detuneMaster = detune + masterPitch;
+		const int transposedKey = key() + masterPitch;
```

Comment:
`auto` would be even better here:
```suggestion
		const auto transposedKey = key() + masterPitch;
```